site/plans.html.jet

```diff
@@ -18,35 +11,57 @@
-          {{ end }}
-            <div class="plan-caption">
-              <div class="plan-name">{{p.Name}}</div>
-              {{ if p.HasExpiryDate() }}
-                <div class="plan-expiry">{{ i18n("plan_expiry_date", map("ExpiryDate", p.ExpiryDate)) }}</div>
-              {{ end }}
-              <div class="plan-description">{{p.Description}}</div>
-            </div>
-
-          {{ if p.Interval }}
-            <div class="plan-details">
-              <div class="plan-interval">{{i18n("shopping_info_plan_offer", map("Interval", p.Interval, "Count", p.IntervalCount))}}</div>
-              {{ if p.TrialPeriodDays }}
-                <div class="plan-trial-period">{{i18n("shopping_info_trial_period_offer", p.TrialPeriodDays)}}</div>
-              {{ end }}
-            </div>
-          {{ end }}
-          <div class="plan-buttons">
-            {{ if p.Page }}
-              <button class="plan-page-link"><a href="{{ routeToSlug(p.Page.Slug) }}">{{i18n("plan_read_more")}}</a></button>
+        <div class="plan col-xs-6 col-lg-4 mb-3">
+          <div class="card h-100">
+            {{ if isset(p.LandscapeImage) && p.LandscapeImage != ""}}
+              <div class="poster poster-landscape">
+                {{ if p.Page }}
+                  <a href="{{ routeToSlug(p.Page.Slug) }}">
+                {{ end }}
+                <s72-image src="{{p.LandscapeImage}}" alt="{{p.Name}}" class="plan-poster card-img-top" border="0"></s72-image>
+                <s72-plan-label data-slug="{{p.Slug}}" data-title="{{p.Name}}"></s72-plan-label>
+                {{ if p.Page }}
+                  </a>
+                {{ end }}
+              </div>
             {{ end }}
-            <s72-pricing-buttons data-slug="{{p.Slug}}" data-title="{{p.Name}}" />
-            <s72-plan-label data-slug="{{p.Slug}}" data-title="{{p.Name}}"/>
+              <div class="card-body d-flex flex-column justify-content-between">
+                <div class="plan-caption-details">
+
+                  {{ if !isset(p.LandscapeImage) || p.LandscapeImage == "" }}
+                    <s72-plan-label data-slug="{{p.Slug}}" data-title="{{p.Name}}"></s72-plan-label>
+                  {{ end }}
+
+                  <h3 class="plan-name">{{p.Name}}</h3>
+
+                  {{ if p.HasExpiryDate() }}
+                    <p class="alert p-0 border-0">
+                      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-exclamation-circle" viewBox="0 0 16 16"> <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/> <path d="M7.002 11a1 1 0 1 1 2 0 1 1 0 0 1-2 0zM7.1 4.995a.905.905 0 1 1 1.8 0l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 4.995z"/> </svg>
+                      <small>
+                        {{ i18n("plan_expiry_date") }} <s72-time datetime="{{ p.ExpiryDate.UnixMilli() }}" />
```

Comment:
FYI, the humanized date based off the browsers language is a current source of headaches with a large number of clients  not happy with the lack of translatableness. Just noting here as it's something we want to change for availability labels so this might also need a change later too.

Reply:
Cool. Not sure if you're aware - BAs are looking into a (manual human) translate service for these files, so that might eliminate some headache in the future.